pom.xml

```
@@ -83,6 +83,7 @@
         <module>test-utils/testcontainers</module>
         <module>test-utils/junit5-extension</module>
         <module>test-utils/assert/core</module>
+        <module>test-utils/assert/qa</module>
```

```suggestion
        <module>test-utils/assert</module>
```

With introducing the parent `pom.xml`, we don't need to call each child module separately, right? We probably need to change it for the sonatype profile as well.